cloud/endagaweb/templates/dashboard/user_management/add.html

```diff
@@ -0,0 +1,354 @@
+<script>
+
+    function changeRole(role) {
+        if (role.value) {
+            $.ajax({
+                url: "/dashboard/user/management/permissions?role=" + role.value
+            }).done(function(response) {
+                moveAllItems('#permissions_right', '#permissions_left');
+
+                $.each(response.permissions, function(key,val){
+
+                    var left = $('#permissions_left option[value="'+val+'"]');
+                    $('#permissions_right').append(left);
+
+                });
+            });
+        } else {
+            moveAllItems('#permissions_right', '#permissions_left');
+        }
+    }
+
+    function moveItems(origin, dest) {
+        $(origin).find(':selected').appendTo(dest);
+        $('.no_perm_error').hide();
+        $('.no_perm_error').parent().removeClass('has-warning');
+    }
+
+    function moveAllItems(origin, dest) {
+        $(origin).children().appendTo(dest);
+        $('.no_perm_error').hide();
+        $('.no_perm_error').parent().removeClass('has-warning');
+    }
+
+    $(document).ready(function() {
+        $("#permissions_row tr:odd").css("background", "#DAE4F0");
+        $("#permissions_row tr:even").css("background", "#eee");
+
+        $('#lstview_rightSelected').click(function () {
+            moveItems('#permissions_left', '#permissions_right');
+        });
+
+        $('#lstview_leftSelected').on('click', function () {
+            moveItems('#permissions_right', '#permissions_left');
+        });
+
+        $('#lstview_leftAll').on('click', function () {
+            moveAllItems('#permissions_right', '#permissions_left');
+        });
+
+        $('#lstview_rightAll').on('click', function () {
+            moveAllItems('#permissions_left', '#permissions_right');
+        });
+
+        // Email validation
+
+        $('form input[name="email"]').blur(function () {
+            $('#email').removeClass('form-control-success');
+            $('#email').removeClass('form-control-warning');
+            var email = $(this).val();
+            var _changeInterval = null;
+            var re = /[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}/igm;
+            if (re.test(email)) {
+
+                $('.msg').hide();
+                $('.success').show();
+                $('.exists').hide();
+                $('#email').parent().parent().removeClass('has-success');
+                $('#email').parent().parent().removeClass('has-warning');
+
+                // Check if email exists...
+
+                clearInterval(_changeInterval);
+                _changeInterval = setInterval(function() {
+                    clearInterval(_changeInterval)
+                    console.log("checking if email exists...");
+                    $.ajax({
+                        url: "/dashboard/user/management/checkuser?email=" + $('#email').val(),
+                        //beforeSend: function(xhr) {
+                        // xhr.setRequestHeader("X-CSRFToken", '{{ csrf_token }}');
```

Comment:
CSRF should be required. I think it gets switched on for POST by default so you can change the request type for this endpoint